charts/prometheus-postgres-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "0.11.1"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 4.3.0
+version: 4.3.1
```

Adding feature should bump to minor version.

```suggestion
version: 4.4.0
```

---

Fixed